openedx/features/caliper_tracking/tests/expected/edx.librarycontentblock.content.removed.json

```diff
@@ -0,0 +1,59 @@
+{
+    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "action": "Deleted",
```

Comment:
Please change the `actor`, `object` and `action` to  suit the use case.

Reply:
done. kindly review the changes now.